.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+#-   repo: https://github.com/PowerShell/PSScriptAnalyzer
+-   repo: https://github.com/juju4/PSScriptAnalyzer
```

Comment:
i'm sorry, but we can't have a pointer to your repository

Reply:
This was just for testing. reverted.
Note that rev will have to be updated after to an appropriate commit/tag/release too.